### PR TITLE
Flashing ELRS initial progress

### PR DESCRIPTION
**Pull request by deadbytefpv**
Here's an initial progress on the Flashing page.

Feel free to add to the page and correct any wrong info.

Thank you!

### CODE REVIEW

**Review by maybenikhil — CHANGES_REQUESTED** [review]
Do we really need to include the info on STLink Flashing? Other than that looks great!

**Comment by deadbytefpv** [comment]
I think we need to include it, those that's already in the know will look for it. It's still a valid update/flashing method.

The Slim(+) still depends on it, as well as the Lite (Pro) tx modules...

If anything, I should've added that it's only used as last resort for the r9mm/mini/mx and r9M..

File: docs/quick-start/flashing-elrs.md [hunk 1]
```diff
@@ -32,18 +77,93 @@ TODO
...
+
+*Note: if you haven't bridged the `Boot` pads but the receiver has solid LED light, your FC is probably pulling the current UART's RX pad `LOW` which will interfere with the normal and passthrough flashing of this receiver. Find another UART and wire your receiver there instead*
+
+Bridging the `Boot` pads is no longer needed past 1.0.0-RC6. 
+
+Power your FC with a LiPo, or if receiver is powered via USB (receiver is connected to a 4v5 pad), connect the FC to your USB port. Using the ExpressLRS Configurator, with the correct Target selected and options set, click on **Build & Flash**. Wait for the process to finish and you should be greeted with the "Success" banner.
+
+Unplug USB and LiPo, and removed the solder on the bridged `Boot` pads. You no longer need it (past 1.0.0-RC6). Power your TX Module and then your FC to verify you are bound and has connection.
+
+#### Happymodel PP RXes
```
Comment (jurgelenas) [inline docs/quick-start/flashing-elrs.md:155]: I think only STM variant PP does not have wifi capabilities? They have ESP32 based ones and they support wifi just fine.

**Review by jurgelenas — CHANGES_REQUESTED** [review]

**Comment by jurgelenas** [comment]
> I think we need to include it, those that's already in the know will look for it. It's still a valid update/flashing method.
> 
> The Slim(+) still depends on it, as well as the Lite (Pro) tx modules...
> 
> If anything, I should've added that it's only used as last resort for the r9mm/mini/mx and r9M..

Agreee, st-link method is still useful.

File: docs/quick-start/flashing-elrs.md (hunk 1 above)
Comment (deadbytefpv) [reply docs/quick-start/flashing-elrs.md:155]: Are there Happymodel PPs that were using ESP MCUs now?

Comment (maybenikhil) [reply docs/quick-start/flashing-elrs.md:155]: I thought PPs were exclusively STM32 yeah...

Comment (jurgelenas) [reply docs/quick-start/flashing-elrs.md:155]: Technically those are called EP1 / EP2.. 😄

Comment (deadbytefpv) [reply docs/quick-start/flashing-elrs.md:155]: But this section is specific for the Happymodel PPs..

**Review by maybenikhil — APPROVED** [review]
lgtm

**Pull request merged**